Docs/source/usage/parameters.rst

```diff
@@ -856,7 +856,7 @@ Particle initialization
     species that impact the embedded boundary.
     The scraped particle buffer can be used to track particle fluxes out of the
     simulation but is currently only accessible via the Python interface. The
-    ``pywarpx`` function ``get_particle_boundary_buffer()`` can be
+    ``pywarpx.libwarpx`` function ``get_particle_boundary_buffer()`` can be
```

Comment:
Feel free to push once more if you like to revise this or add a note on where users should get it from (sim object)

Reply:
@ax3l just pushed, good catch!